### ISSUE TITLE
Nodebreaker Outputs to Front Instead of Back in Survival Mode

### ISSUE THREAD

**Issue by minewizard1962** [mt-mods/pipeworks#131]
Nodebreaker in Survival Mode Outputs Block to Front Instead of Back
When playing in Survival mode, the Nodebreaker from Pipeworks outputs blocks to the front of the machine instead of the back. This issue does not occur in Creative mode. 
To reproduce the bug, follow these steps:
Start a VoxeLibre (formerly MineClone2) server.
Setup a Nodebreaker in your world.
Enter Survival mode.
Use the Nodebreaker to break a block. Observe that the outputted block appears in front of the Nodebreaker.
Expected Behavior:
The Nodebreaker should output the broken block to the back of the machine, regardless of the game mode. 
Environment:
Minetest version 5.8.0
Additional Information:
This bug only occurs in Survival mode. In Creative mode, the Nodebreaker works as intended. 
![creative_mode](https://github.com/user-attachments/assets/f096d4de-c075-417b-a03e-7559f76cdc04)
![survival_mode](https://github.com/user-attachments/assets/e5df3511-c0b1-455f-ba1b-0d0d19c68158)

**Comment by OgelGames** [comment]
That's weird, wielders should never eject items from the front. Here's the code responsible for ejecting items, for reference: https://github.com/mt-mods/pipeworks/blob/e9a9bd711ecef35b36d319e3ff9c9f81e17084a6/wielder.lua#L54-L77

**Comment by SwissalpS** [comment]
I can confirm that this happens with VoxelLibre downloaded recently from CDB and pipeworks from before fakelib was added.
Same occurs using Mineclonia as base game.
Was not able to recreate with Hades or MTG based games.

Possibly this has to do with how the MC family delivers drops? MTG and Hades put dug items straight into player's inventory while MC games drop them into world for players to 'vacuum' up.

**Comment by OgelGames** [comment]
> Possibly this has to do with how the MC family delivers drops? MTG and Hades put dug items straight into player's inventory while MC games drop them into world for players to 'vacuum' up.

Oh, that's exactly what it is, I didn't even think about that, the items never even get put in the nodebreaker.

**Comment by OgelGames** [comment]
The item drop mod probably also causes the same bug: https://content.minetest.net/packages/texmex/item_drop/

**Comment by SwissalpS** [comment]
Slightly related:
In the MC-games, deployers only accept items from tubes from the back. Possibly considered a feature ;)

**Comment by OgelGames** [comment]
> deployers only accept items from tubes from the back. Possibly considered a feature

Yes, it's been like that for a long time, in MTG too: https://github.com/mt-mods/pipeworks/blob/e9a9bd711ecef35b36d319e3ff9c9f81e17084a6/wielder.lua#L306
